src/components/Modal/index.tsx

```diff
@@ -184,29 +185,38 @@ const Body = ({ children, withoutPadding = false }: BodyProps): ReactElement =>
 )
 
 /*** Footer ***/
-const FooterSection = styled.div`
+const FooterSection = styled.div<{ withoutBorder: boolean }>`
   display: flex;
   justify-content: center;
-  border-top: 2px solid ${({ theme }) => theme.colors.separator};
-  padding: 24px;
+  align-items: center;
+  border-top: ${({ withoutBorder }) => (withoutBorder ? '0' : '2px')} solid ${({ theme }) => theme.colors.separator};
+  height: 84px;
+  gap: 16px;
 `
 
-const ButtonStyled = styled(Button)`
-  &.MuiButtonBase-root {
-    margin: 0 10px;
-  }
+const LoaderText = styled.span`
+  margin-left: 10px;
 `
 
+export enum ButtonStatus {
+  ERROR = -1,
```

Comment:
Why only the error status has a numeric value?

Reply:
ah, to start counting the enum from `-1` instead of `0`.

![image](https://user-images.githubusercontent.com/3315606/118007863-eceac180-b322-11eb-8b29-18d777ed7fe6.png)
https://tsplay.dev/mplB6m

But if it's unnecessarily confusing, I can get rid of the `= -1` part.